site/_posts/2023-03-10-growing-is-hard.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: blog-post
+title: Building a team is hard&#58; part I
+description: Building a team is harder than it looks. The attrition problem.
+author: Daniel DeGroff
+category: article
+image: blogs/consents-app/managing-consent-header.png
```

Comment:
Assuming "we'll need a graphic" means this is a placeholder.

Reply:
Yes. Sean will help me with a better graphic, this is just a placeholder.